tests/Aspire.Confluent.Kafka.Tests/MetricsTests.cs

```diff
@@ -0,0 +1,345 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.Metrics;
+using System.Threading.Channels;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.Metrics.Testing;
+using Microsoft.Extensions.Hosting;
+using Xunit;
+
+namespace Aspire.Confluent.Kafka.Tests;
+
+public class MetricsTests
+{
+    [ConditionalTheory]
```

Comment:
```suggestion
    [Theory]
```

For all the tests in this file.

Reply:
done